glutin/src/context.rs

```diff
@@ -56,6 +56,9 @@ pub trait NotCurrentGlContext: Sealed {
     /// # Platform specific
     ///
     /// **macOS:** - **This will block if your main thread is blocked.**
+    /// **Wayland:** - This call may latch(will do with mesa drivers) the
+    /// underlying back buffer, meaning that all resize operations will
+    /// apply after the next [`GlSurface::swap_buffers`].
```

Comment:
```suggestion
    /// **Wayland:** - This call may latch the underlying back buffer (will do with mesa drivers), meaning that all resize operations will apply after the next [`GlSurface::swap_buffers`].
```

I think switching it around like this is a little better. It doesn't interrupt the reading process as much because the parenthesis are right in front of a comma anyway. Same applies to the other doc comments of this structure.